src/server/resolvers.js

```diff
@@ -235,10 +235,13 @@ const getResolver = (esConfig, esInstance) => {
   }, {});
 
   const mappingResolvers = esConfig.indices.reduce((acc, cfg) => {
-    log.debug(`${cfg.index} `, esInstance.getESFields(cfg.index));
```

Comment:
why removing this?

Reply:
ah, I added a bunch of debugging statements while I was working and then removed them before pushing. I guess I got too overzealous in removing debugging statements. I'll put this one back 😄